test/controllers/redirect.controller.test.ts

```diff
@@ -284,4 +287,40 @@ describe('RedirectController', () => {
         .expect('Location', SEARCH_COMPANY_URI)
     })
   })
+
+  describe('Unauthorised user', () => {
```

Comment:
I think these should be placed in the describe blocks above i.e. Paid, Pending Payment etc.

Also can we use consistent terminology? Instead of using unauthorised in the tests and not selected signatory in the impl